rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -614,11 +614,58 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
         Send(ChanVar(BoundVar(2)), List[Par](GInt(7)), false, BitSet(2))
           .prepend(Send(ChanVar(BoundVar(1)), List[Par](GInt(8)), false, BitSet(1)))
           .prepend(Send(ChanVar(BoundVar(0)), List[Par](GInt(9)), false, BitSet(0))),
+        Vector.empty,
         BitSet()
       )))
     result.knownFree should be(inputs.knownFree)
   }
 
+  "PNew" should "Sort URI's and place them at the end" in {
```

Comment:
I don't follow -- we're sorting the uris in lexicographical order in the `normalize.scala` instead of `NameSortMatcher` ?

Reply:
We can't sort them in the sorter, than the variable numbering would be messed up.